store/src/directory.rs

```diff
@@ -392,21 +812,80 @@ where
                 Ok(v) if v.is_file() => {}
                 Ok(_) => continue,
             }
-            let ttl = match xattr::get(&path, format_xattr(crate::MetadataKey::<MKT>::Ttl.to_key()))
-            {
-                Err(e) => {
-                    log::trace!("Error looking up TTL xattr for {}: {:?}", path.display(), e);
+            // skip non-metadata files if xattr is not supported
+            if !self.xattr_enabled {
+                match path.as_path().extension() {
+                    None => continue,
+                    Some(ex) => {
+                        if ex != FDO_METADATA_EX {
+                            continue;
+                        }
+                    }
+                }
```

Comment:
metadata mgmt shouldn't filter here, we still want to "play" on OVs, the metadata part should be hidden

Reply:
In order to perform the maintenance we need to get the ttl and that's on the metadata file, that's why need need those